### ISSUE TITLE
Python 2.6 errors

### ISSUE THREAD

**Issue by atward** [delfick/delfick_logging#1]
When using delfick_app from bespin

```
Something unexpected happened!! Please file a ticket in the issue tracker! None
===============================================================================

Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.6.9/bin/bespin", line 9, in <module>
    load_entry_point('bespin==0.7.0.2', 'console_scripts', 'bespin')()
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/delfick_app.py", line 188, in main
    app.mainline(argv, **execute_args)
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/delfick_app.py", line 257, in mainline
    , syslog=args_obj.syslog, syslog_address=args_obj.syslog_address
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/delfick_app.py", line 295, in setup_logging
    , logging_handler_file=self.logging_handler_file
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/delfick_logging.py", line 172, in setup_logging
    handler.setFormatter(SimpleFormatter("{0} {1}".format("%(asctime)s %(levelname)-7s", base_format)))
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/delfick_logging.py", line 16, in __init__
    super(SimpleFormatter, self).__init__(*args, **kwargs)
TypeError: super() argument 1 must be type, not classobj
```

**Comment by delfick** [comment]
Let's just ignore python2.6, there isn't much point in catering for it.

**Comment by atward** [comment]
Looking at commit log I think this may already be fixed. Will test with updated delfick_app

**Comment by atward** [comment]
Appears to be fixed in 5207b3997059dcd1ce0463434587226f741e085b. Update to 0.2.3
